src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -49,3 +49,64 @@ antlrcpp::Any SparqlQleverVisitor::processIriFunctionCall(
   }
   throw ParseException{"Function \"" + iri + "\" not supported"};
 }
+
```

Comment:
The convention (see above) in general is to put a line with a comment of underscores before the implementation,
s.t. the start of a new function becomes more visible:
```
// ____________________________________________________________________________________
SelectClause SparqlQleverVisistor::somemethod() {
...
}
```